subsys/kafka/src/main/java/org/commonjava/indy/subsys/kafka/handler/MetadataServiceKafkaProducer.java

```diff
@@ -0,0 +1,66 @@
+package org.commonjava.indy.subsys.kafka.handler;
+
+import com.fasterxml.jackson.databind.ser.std.StringSerializer;
+
+import org.apache.kafka.clients.producer.KafkaProducer;
+import org.apache.kafka.clients.producer.ProducerConfig;
+import org.apache.kafka.clients.producer.ProducerRecord;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.annotation.PostConstruct;
+import javax.annotation.PreDestroy;
+import javax.enterprise.context.ApplicationScoped;
+
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.Properties;
+
+
+@ApplicationScoped
+public class MetadataServiceKafkaProducer implements KafkaEventProducer {
```

Comment:
Do we need this specific MetadataServiceKafkaProducer? I think if we have the KafkaEventProducer ready, we just need to pass the different event and target topic.